app/plugin_statuses.go

```diff
@@ -70,7 +70,7 @@ func (a *App) GetClusterPluginStatuses() (model.PluginStatuses, *model.AppError)
 	return pluginStatuses, nil
 }
 
-func (a *App) notifyPluginStatusesChanged() error {
+func (a *App) notifyPluginStatusesChanged() *model.AppError {
```

Comment:
nit: why switching back to `*model.AppError`? (In some mythical future, I'm hoping we converge on `error` everywhere.)

Reply:
i hope we switch to using `error` as opposed to `*model.AppError` as well. This was more to be consistent with rest of the error handling.